config/routes.rb

```diff
@@ -37,6 +37,12 @@
 
   resources :categories, only: [:show], param: :category
 
+  namespace :user do
+    resources :stores, only: [:index]
+  end
+
+  resources :stores, only: [:index, :new, :create]
```

Comment:
I notice that the stores controller doesn't have an index action. What's the reasoning for opening up this route?

Reply:
We added this in because the form for creating a new store executes a POST to the stores index.

Reply:
Yep, yep. Thanks for the quick responses to these questions!